wgpu-core/src/command/transfer.rs

```diff
@@ -286,14 +287,15 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
             .surface_desc()
             .bits as u32
             / BITS_PER_BYTE;
-        let buffer_width = destination.layout.bytes_per_row / bytes_per_texel;
+        assert_eq!(wgt::COPY_BYTES_PER_ROW_ALIGNMENT % bytes_per_texel, 0);
```

Comment:
Is this strictly necessary? The value for bytes_per_texel seem to be 1, 2, 4, 8, 16.. which would make this always true.

Reply:
I was hesitating to put this in, since I agree this always succeeds today, but figured that an assert is the best enforcement we can get here, instead of guessing down the line what goes wrong.

Reply:
Fair enough